helm/charts/radius/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "radius.fullname" . }}
+data:
+  LB_ADDR: {{ .Values.configMap.lbAddr }}
+  GLUU_OXAUTH_BACKEND: {{ .Values.configMap.oxauthBackendUrl | quote }}
+  GLUU_CONFIG_ADAPTER: {{ .Values.global.configAdapterName | quote }}
+  GLUU_SECRET_ADAPTER: {{ .Values.global.configSecretAdapter | quote }}
+  DOMAIN: {{ .Values.global.domain | quote }}
+  GLUU_MAX_RAM_FRACTION: {{ .Values.global.gluuMaxRamFraction | quote }}
+  GLUU_PERSISTENCE_TYPE: {{ .Values.global.gluuPersistenceType | quote }}
+  # options: default/user/site/cache/statistic used only if GLUU_PERSISTENCE_TYPE is hybrid
+  {{- if eq .Values.global.gluuPersistenceType "hybrid" }}
+  GLUU_PERSISTENCE_LDAP_MAPPING: {{ .Values.global.gluuPersistenceLdapMapping | quote }}
+  {{- end }}
+  {{- if or (eq .Values.global.gluuPersistenceType "ldap") (eq .Values.global.gluuPersistenceType "hybrid")  }}
+  # used only if GLUU_PERSISTENCE_TYPE is ldap or hybrid
+  GLUU_LDAP_URL: {{ .Values.global.gluuLdapUrl | quote }}
+  {{- else if or (eq .Values.global.gluuPersistenceType "couchbase") (eq .Values.global.gluuPersistenceType "hybrid") }}
+  #  used only if GLUU_PERSISTENCE_TYPE is couchbase or hybrid
```

Comment:
We can discuss CB later but most these envs need to be also be added to the other configmaps charts.